spectral_cube/dask_spectral_cube.py

```diff
@@ -310,6 +310,45 @@ def _get_filled_data(self, view=(), fill=np.nan, check_endian=None, use_memmap=N
         else:
             return da.from_array(FilledArrayHandler(self, fill=fill), name='FilledArrayHandler ' + str(uuid.uuid4()), chunks=data.chunksize)[view]
 
+    def __repr__(self):
+        default_repr = super().__repr__()
+        lines = default_repr.splitlines()
+        lines[0] = lines[0][:-1] + ' and chunk size {0}:'.format(self._data.chunksize)
+        return '\n'.join(lines)
+
+    @add_save_to_tmp_dir_option
+    def rechunk(self, chunks='auto', threshold=None, block_size_limit=None,
+                **kwargs):
+        """
+        Rechunk the underlying dask array and return a new cube.
```

Comment:
A link to the dask array chunks docs here would be useful for more explanation of the kwargs. `threshold` seems hard to understand without some understanding of what dask is doing.

https://docs.dask.org/en/latest/array-chunks.html

Reply:
Done